Source/JavaScriptCore/wasm/js/JSWebAssemblyArray.h

```diff
@@ -58,7 +58,12 @@ class JSWebAssemblyArray final : public WebAssemblyGCObjectBase {
 
     static JSWebAssemblyArray* create(VM& vm, Structure* structure, Wasm::FieldType elementType, unsigned size, RefPtr<const Wasm::RTT>&& rtt)
```

Comment:
If it fails, function name should be tryCreate.

Reply:
Oops, thanks.